libs/entitlement/agent/build.gradle

```diff
@@ -6,51 +6,18 @@
  * your election, the "Elastic License 2.0", the "GNU Affero General Public
  * License v3.0 only", or the "Server Side Public License, v 1".
  */
-
-import static java.util.stream.Collectors.joining
-
 apply plugin: 'elasticsearch.build'
-apply plugin: 'elasticsearch.embedded-providers'
-
-embeddedProviders {
-  impl 'entitlement-agent', project(':libs:entitlement:agent:impl')
-}
-
-configurations {
-  entitlementBridge
-}
 
 dependencies {
-  entitlementBridge project(":libs:entitlement:bridge")
   compileOnly project(":libs:core")
   compileOnly project(":libs:entitlement")
-  testImplementation project(":test:framework")
-  testImplementation project(":libs:entitlement:bridge")
-  testImplementation project(":libs:entitlement:agent:impl")
-}
-
-tasks.named('test').configure {
```

Comment:
I've deleted the tests that required this. We consider them unnecessary now, since practically nothing is actually in the agent; the logic that used to be there is in the Entitlement library now and can be tested independently.